tests/e2e/tablespaces_test.go

```diff
@@ -116,9 +158,91 @@ var _ = Describe("Tablespaces tests", Label(tests.LabelSmoke, tests.LabelBasic),
 		})
-	})
 
-	Context("plain cluster", Ordered, func() {
+		It("can create the backup and verify content in the object store", func() {
+			backupName, err = env.GetResourceNameFromYAML(clusterBackupManifest)
+			Expect(err).ToNot(HaveOccurred())
+
+			By(fmt.Sprintf("creating backup %s and verifying backup is ready", backupName), func() {
+				testUtils.ExecuteBackup(namespace, clusterBackupManifest, false, testTimeouts[testUtils.BackupIsReady], env)
+				AssertBackupConditionInClusterStatus(namespace, clusterName)
+			})
+
+			By("verifying the number of tars in minio", func() {
+				backupTars := minioPath(clusterName, "*.tar")
+				// as there are 2 tablespaces, we should have 3 tar files (data.tar + one tar per tablespace)
+				Eventually(func() (int, error) {
+					return testUtils.CountFilesOnMinio(namespace, minioClientName, backupTars)
+				}, 60).Should(BeEquivalentTo(3))
+			})
+
+			By("verifying backup status", func() {
+				Eventually(func() (string, error) {
+					cluster, err := env.GetCluster(namespace, clusterName)
+					return cluster.Status.FirstRecoverabilityPoint, err
+				}, 30).ShouldNot(BeEmpty())
+				Eventually(func() (string, error) {
+					cluster, err := env.GetCluster(namespace, clusterName)
+					return cluster.Status.LastSuccessfulBackup, err
+				}, 30).ShouldNot(BeEmpty())
+				Eventually(func() (string, error) {
+					cluster, err := env.GetCluster(namespace, clusterName)
+					return cluster.Status.LastFailedBackup, err
+				}, 30).Should(BeEmpty())
+			})
+		})
+
+		It("can update the cluster adding a new tablespace and backup again", func() {
+			By("adding a new tablespace to the cluster", func() {
+				cluster, err := env.GetCluster(namespace, clusterName)
+				Expect(err).ToNot(HaveOccurred())
+
+				addTablespaces(cluster, map[string]*apiv1.TablespaceConfiguration{
+					"thirdtablespace": {
+						Storage: apiv1.StorageConfiguration{
+							Size: "1Gi",
+						},
+					},
+				})
+
+				cluster, err = env.GetCluster(namespace, clusterName)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(cluster.ShouldCreateTablespaces()).To(BeTrue())
+			})
+			By("waiting for the cluster to be ready", func() {
+				AssertClusterIsReady(namespace, clusterName, testTimeouts[testUtils.ClusterIsReady], env)
+			})
+
+			By(fmt.Sprintf("creating new backup %s and verifying backup is ready", backupName), func() {
+				testUtils.ExecuteBackup(namespace, clusterBackupManifest, false, testTimeouts[testUtils.BackupIsReady], env)
+				AssertBackupConditionInClusterStatus(namespace, clusterName)
+			})
+
+			By("verifying the number of tars in minio", func() {
+				backupTars := minioPath(clusterName, "*.tar")
+				// as there are 2 tablespaces, we should have 3 tar files (data.tar + one tar per tablespace)
+				Eventually(func() (int, error) {
+					return testUtils.CountFilesOnMinio(namespace, minioClientName, backupTars)
+				}, 60).Should(BeEquivalentTo(3))
```

Comment:
This is surprising to me. At this point we should have 3 tablespaces, and therefore 4 tar files, not 3.

Reply:
yes, it should be 4